packages/pentaho/src/Canvas/PanelTab/PanelTab.styles.tsx

```diff
@@ -12,7 +12,7 @@ export const { staticClasses, useClasses } = createClasses("HvCanvasPanelTab", {
     alignItems: "center",
     justifyContent: "center",
     flex: 1,
-    borderRadius: "16px 16px 0 0",
+    borderRadius: `${theme.radii.large} ${theme.radii.large} 0 0`,
```

Comment:
this would be (which I'm not suggesting, please don't 😅 ): 
```
borderRadius: `${theme.radii.large} ${theme.radii.large} ${theme.radii.none} ${theme.radii.none}`,
```